service/collector/cluster_info.go

```diff
@@ -0,0 +1,68 @@
+package collector
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	clusterLabel           = "cluster_id"
+	statusLabel            = "status"
+	listClusterConfigLimit = 20
+)
+
+var (
+	clustersDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "cluster_info"),
+		"Cluster information.",
+		[]string{
+			clusterLabel,
+			statusLabel,
+		},
+		nil,
+	)
+)
```

Comment:
Nitpick: why not just

```go
var  clustersDesc *prometheus.Desc = prometheus.NewDesc(
	prometheus.BuildFQName(Namespace, "", "cluster_info"),
	"Cluster information.",
	[]string{
		clusterLabel,
		statusLabel,
	},
	nil,
)
```

I personally don't see a reason to align such initializations.